core/gui/src/app/workspace/service/workflow-status/operator-reuse-cache-status.service.ts

```diff
@@ -19,41 +13,9 @@ export class OperatorReuseCacheStatusService {
     private workflowWebsocketService: WorkflowWebsocketService,
```

Comment:
Can we just remove this class completely? Originally it is used by the EditingTimeCompilation, but now it is unused in GUI.

Reply:
OperatorReuseCacheStatusService

Reply:
kept the OperatorReuseCacheStatusService.